examples/hosted-wallet-kt/pfi/src/main/kotlin/api/Exchanges.kt

```diff
@@ -30,23 +30,23 @@ class Exchanges(private val bearerDid: BearerDid, private val offeringsRepositor
         val replyTo = requestBody.replyTo ?: throw Exception("replyTo cannot be null for this example")
         val rfq = requestBody.message
 
+        rfq.verify()
         rfq.verifyOfferingRequirements(this.offeringsRepository.getOffering(rfq.data.offeringId))
 
         this.exchangesToReplyTo[rfq.metadata.exchangeId] = replyTo
 
         res.status(202)
 
         Thread {
-            Thread.sleep(3000)
+            Thread.sleep(500)
```

Comment:
you don't think this could intermittently fail on a slower cloud box or something?

Reply:
yeah probably true, but meh